samples/sampler/stories/default/Calendar.js

```diff
@@ -10,13 +11,41 @@ export default {
 	component: 'Calendar'
 };
 
-export const _Calendar = (args) => (
-	<Calendar
-		disabled={args['disabled']}
-	/>
-);
+const days = [6, 12, 15, 20, 22, 28];
+const months = [
+	'January',
+	'February',
+	'March',
+	'April',
+	'May',
+	'June',
+	'July',
+	'August',
+	'September',
+	'October',
+	'November',
+	'December'
+];
+const years = [2021, 2022, 2023, 2024, 2025, 2026];
+
+export const _Calendar = (args) => {
+	const [day, setDay] = useState(new Date(args['year'], months.indexOf(args['month']), args['day']));
+
+	useEffect(() => setDay(new Date(args['year'], months.indexOf(args['month']), args['day'])), [args]);
+
+	return (
+		<Calendar
+			disabled={args['disabled']}
+			selectedDate={day}
+			setSelectedDate={setDay}
+		/>
+	);
+};
 
 boolean('disabled', _Calendar, Config);
+select('day', _Calendar, days, Config, 6);
+select('month', _Calendar, months, Config, 'February');
+select('year', _Calendar, years, Config, 2025);
```

Comment:
please add an action to show current selected date

Reply:
Done